app/Models/Contest.php

```diff
@@ -221,7 +262,7 @@ public function defaultJson($user = null)
 
         return json_encode([
```

Comment:
on unrelated note, apparently none of other `defaultJson` actually return json. Also none of other json_encode explicitly set the options.